rivery_cli/converters/entities/rivers.py

```diff
@@ -300,7 +300,9 @@ def step_importer(cls, steps: list, rivery_session: RiverySession, code_dir: str
                 current_step["loop_over_value"] = step.pop("loop_over_value", "")
                 current_step["loop_over_variable_name"] = step.pop("loop_over_variable_name", [])
                 current_step["steps"] = cls.step_importer(
-                    steps=step.pop('nodes', [])
+                    steps=step.pop('nodes', []),
+                    rivery_session=rivery_session,
+                    code_dir=code_dir
```

Comment:
why there's a need on rivery_session there?

Reply:
@Alonreznik  it was this way in the initial commit, just changed it, please take another look